api_documentation/Core.swagger_collection.json

```diff
@@ -1425,6 +1433,42 @@
         }
       }
     },
+    "/containers/{container-id}/import/datasources/{dataSourceID}/file": {
+      "post": {
+        "description": "Uploads a file and metadata to Deep Lynx",
+        "summary": "Upload File",
+        "tags": [
+          "Data Sources"
+        ],
+        "operationId": "UploadFile",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "container-id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": ""
+          },
+          {
+            "name": "dataSourceID",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": ""
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Do we want the response to be a 201, with it being a POST and a resource being created?

Reply:
Not sure how to change this tbh. This is the swagger documentation which is autogenerated by the Postman documentation. Apart from manually editing the file, which I don't know what havoc that will cause, I don't know how to change it.

Reply:
Ahh, gotcha. There are ways with C#, I don't know how to do it with node.